gc/base/GCExtensionsBase.hpp

```diff
@@ -1443,6 +1443,16 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	bool isSATBBarrierActive();
 	bool usingSATBBarrier();
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/**
+	 * Helper function to determine whether a GC reinitialization is taking place
+	 * as a result of a VM snapshot restore.
+	 * @param[in] env the current environment.
```

Comment:
function description is inaccurate since the method does not accept any input parameters.